src/test/java/checker/MiLoGCheckerDayTimeExceedanceTest.java

```diff
@@ -0,0 +1,261 @@
+package checker;
+
+import static org.junit.Assert.*;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.time.YearMonth;
+import java.util.Random;
+
+import org.junit.Test;
+
+import data.Employee;
+import data.Entry;
+import data.Profession;
+import data.TimeSheet;
+import data.TimeSpan;
+import data.WorkingArea;
+
+public class MiLoGCheckerDayTimeExceedanceTest {
+    
+    //// Placeholder for random tests
+    //Exclusively. Refer to https://docs.oracle.com/javase/8/docs/api/java/util/Random.html
+    private static final int RANDOM_HOUR_BOUND = 24;
+    private static final int RANDOM_MINUTES_BOUND = 60;
+    
+    //// Placeholder for time sheet construction
+    private static final Employee EMPLOYEE = new Employee("Max Mustermann", 1234567);
+    private static final Profession PROFESSION = new Profession("Fakultät für Informatik", WorkingArea.UB, new TimeSpan(40, 0), 10.31);
+    private static final YearMonth YEAR_MONTH = YearMonth.of(2019, Month.NOVEMBER);
+    private static final TimeSpan ZERO_TS = new TimeSpan(0, 0);
+    private static final LocalDate WORKINGDAY_VALID = LocalDate.of(2019, 11, 22);
+
+    @Test
+    public void testUpperBoundValidSingleEntry() throws CheckerException {
+        ////Test values
+        TimeSpan start = new TimeSpan(8, 0);
+        TimeSpan end = new TimeSpan(18, 45);
+        TimeSpan pause = new TimeSpan(0, 45);
+
+        ////Checker initialization
+        Entry entry = new Entry("Test", WORKINGDAY_VALID, start, end, pause);
+        Entry[] entries = {entry};
+        TimeSheet timeSheet = new TimeSheet(EMPLOYEE, PROFESSION, YEAR_MONTH, entries, ZERO_TS, ZERO_TS, ZERO_TS);
+        MiLoGChecker checker = new MiLoGChecker(timeSheet);
+
+        ////Execution
+        checker.checkDayTimeExceedance();
+
+        ////Expectation
+        String error = String.format(MiLoGChecker.CheckerErrorMessage.DAY_TIME_EXCEEDANCE.getErrorMessage(), 
+                MiLoGChecker.getWorkdayMaxWorkingTime(), WORKINGDAY_VALID);
```

Comment:
This not only tests the check method, but also the construction of the error message. I don't know if there is a better solution for now, but I'm unhappy with the error construction in general, so maybe we should address that in a separate PR.